railgun/manager/controllers.go

```diff
@@ -0,0 +1,252 @@
+package manager
```

Comment:
Trying to understand what problem this PR is trying to resolve/address: one typical/classic controller.go under an operator would have following functions 
- New() 
- Reconcile ()
- Enqueue()
If you name it as "controllers.go", assuming we are bumping all controller in a central place, but still, we are just tune configuration - configure controller (enablement or not). 
This refactor brings in confusing.

Reply:
> Trying to understand what problem this PR is resolve/address

Please see the PR description: this is all non-functional changes. Let me know if you feel the description needs to be improved.

> one typical/classic controller.go under an operator would have following functions
> New()
> Reconcile ()
> Enqueue()

These may be typical of a controller implementation, but this is is not a controller implementation this is for the manager runtime.

> If you name it as "controllers.go", assuming we are bumping all controller in a central place, but still, we are just tune configuration - configure controller (enablement or not).
This refactor brings in confusing.

Would you consider this resolved if I renamed this back to `controllerdef.go`, or do you have another name in mind?

Reply:
Changed the naming back to `controllerdef` as it previously was: c2b24bcc287433ae6c019fe750fdcd371bad4dc0